apps/rush-lib/src/start.ts

```diff
@@ -1,6 +1,25 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
+import * as colors from 'colors';
+import * as semver from 'semver';
+
+// check to ensure that we are using a supported version of NodeJS, otherwise write a warning
+if (semver.satisfies(process.versions.node, '< 6.0.0')) {
```

Comment:
>6.0.0 [](start = 47, length = 5)

test this with Daron's 5.6.0 or whatever seems like the oldest sane version.

For even older stuff, you could add an "engines" constraint in package.json